app/javascript/components/editor-config/citations.js

```diff
@@ -288,27 +293,34 @@ function renderPostLinks(postLinks, dispatch, state) {
   return node
 }
 
-function ThreadedPostLink(props) {
-  const { postLink, dispatch, state } = props
+function ThreadedCitation(props) {
+  const { citation, dispatch, state } = props
 
   const handleDelete = () => {
     dispatch(
-      state.tr.setMeta(postLinkPlugin, { type: 'deletePostLink', postLink })
+      state.tr.setMeta(citationPlugin, { type: 'deleteCitation', citation })
     )
   }
 
   var { currentUser } = store.getState()
+
+  const sanitizedTitle = parse(
```

Comment:
Please replace this with the `sanitizeHtml` library we already use, e.g. on `PostPage`, and remove this library from our packages.

Reply:
`sanitizeHtml` doesn't solve the issue, so keeping `html-react-parser`.